src/components/NavBar.vue

```diff
@@ -15,6 +15,7 @@
                 </b-nav-item-dropdown>
                 <b-nav-item-dropdown text="Community">
                     <b-dropdown-item to="/community/">The Galaxy Community</b-dropdown-item>
+                    <b-dropdown-item to="/get-started/">Get started</b-dropdown-item>
```

Comment:
@NickSto we are editing here the NavBar, and I guess that the tests won't pass because of that. Could you help us? Thanks!

Reply:
Looks like the tests are still passing. All the NavBar looks at is the "Community" drop-down, the "Governance" link under that, and the "Training" link.

The only thing that's failing right now is the linter.